common/app/views/support/GetClasses.scala

```diff
@@ -58,7 +58,7 @@ object GetClasses {
       containerDefinition.showLatestUpdate,
       containerDefinition.index == 0 && containerDefinition.customHeader.isEmpty,
       containerDefinition.displayName.isDefined,
-      containerDefinition.displayName == Some("headlines"),
+      containerDefinition.displayName.contains("headlines"),
```

Comment:
See https://github.com/guardian/frontend/blob/master/project/Prototypes.scala#L26

You're behind the times!